client/src/main.jsx

```diff
@@ -1,10 +1,12 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './index.css'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App.jsx";
+import AdminLoginTest from "./AdminLoginTest.jsx";
+import "./index.css";
 
-ReactDOM.createRoot(document.getElementById('root')).render(
+ReactDOM.createRoot(document.getElementById("root")).render(
   <React.StrictMode>
+    <AdminLoginTest />
```

Comment:
Recommend this default to being commented out so that others must explicitly uncomment when pulled to local in order to test. That way a conscious effort to test this 'test' feature must be made so we are not conditioned to the test component being included in production.